examples/extended/grails-app/views/register/secuirtyQuestions.gsp

```diff
@@ -0,0 +1,19 @@
+<html>
```

Comment:
The name of this view is mispelled.

Instead of `secuirtyQuestions` it should be `securityQuestions`.

This should also be corrected anywhere it is referenced.

Reply:
Will Fix